### DISCUSSION TITLE
Zigbee2MQTT errore avvio

### DISCUSSION THREAD

**Discussion by danesima** [indomus/forum#2738] [Networking]
Salve a tutti, ormai sono diversi giorni che sto impazzendo per risolvere questo errore che mi da il componente Zigbee2MQTT su Homeassistant HASSIO (Raspberry PI3b+). Il messaggio di errore all'avvio da registore è questo:

```
[14:25:20] INFO: Preparing to start...
[14:25:22] INFO: Socat not enabled
[14:25:26] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-11-30 14:25:50: Logging to console and directory: '/config/zigbee2mqtt/log/2023-11-30.14-25-50' filename: log.txt
Zigbee2MQTT:info  2023-11-30 14:25:51: Starting Zigbee2MQTT version 1.33.2 (commit #unknown)
Zigbee2MQTT:info  2023-11-30 14:25:51: Starting zigbee-herdsman (0.21.0)
Zigbee2MQTT:error 2023-11-30 14:25:52: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-11-30 14:25:52: Failed to start zigbee
Zigbee2MQTT:error 2023-11-30 14:25:52: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-11-30 14:25:52: Exiting...
Zigbee2MQTT:error 2023-11-30 14:25:52: Error: spawn udevadm ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:285:19)
    at onErrorNT (node:internal/child_process:485:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

Come coordinator ho un ITEAD SONOFF Zigbee 3.0 USB Dongle Plus V2 con firmware NCP 

Altra informazione è che Mosquitto broker si avvia correttammente (o almeno credo), ma dopo l'avvio fallace di Zigbee2Mqtt, anche il componente Mosquitto broker si arresta.

Grazie per l'attenzione. Se necessario vi posso fornire di tutte le informazioni che occorrono.

Matteo

**Comment by snakuzzo** [comment]
https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/420

**Comment by indomus** [comment]
> Zigbee2MQTT:error 2023-11-30 14:25:52: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions

Zigbee2MQTT:error 2023-11-30 14:25:52: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
